src/app/app-routing.module.ts

```diff
@@ -13,10 +13,11 @@ const routes: Routes = [
   //{path:'',component: AppComponent,pathMatch: 'full'},
   {path: '' , component: IndexComponent},
   {path:'itinerarios/:id', component: ItinerarylistComponent,pathMatch: 'full'},
-  {path:'mis_itinerarios/:id', component: MyItinirariesComponent,pathMatch: 'full'},
+  {path:'mis_itinerarios/:id', component: MyItinerariesComponent,pathMatch: 'full'},
   {path:'itinerario/:id', component: ItineraryViewContoller,pathMatch: 'full'},
   {path: 'login', component: LoginComponent},
   {path: 'register', component: RegisterComponent},
+  {path: 'error', component: NotFoundComponent},
```

Comment:
Codacy has a fix for the issue: [Strings must use doublequote.](https://app.codacy.com/gh/ispp-yourney/angular-app/pullRequest?prid=7181827)
```suggestion
  {path: "error", component: NotFoundComponent},
```